components/link/test/link.vdiff.js

```diff
@@ -0,0 +1,61 @@
+import '../link.js';
+import { expect, fixture, focusElem, html } from '@brightspace-ui/testing';
+
+describe('d2l-link', () => {
+
+	before(async() => {
+		return new Promise(resolve => {
+			const link = document.createElement('link');
+			link.id = 'link-sass';
+			link.type = 'text/css';
+			link.rel = 'stylesheet';
+			link.href = './test/sass.output.css';
+			link.onload = resolve;
+			document.getElementsByTagName('head')[0].appendChild(link);
```

Comment:
This was the solution I landed on to get the Sass generated CSS into the page. It's not particularly amazing, but I'm also hoping doing something like this will be rare -- especially outside of core.